cmd/gapir/cc/main.cpp

```diff
@@ -560,6 +560,11 @@ void android_process(struct android_app* app, int32_t cmd) {
       GAPID_DEBUG("Received window: %p\n", gapir::android_window);
       break;
     }
+    case APP_CMD_PAUSE: {
+      GAPID_WARNING("EXITING BECAUSE OF APP_CMD_PAUSE. GAPIR DOES NOT WORK CORRECTLY FROM PAUSE.");
+      exit(0);
```

Comment:
Can we do a cleaner shutdown here including calling `server->shutdown()` and `ANativeActivity_finish()`?

Reply:
There is already exit(0) in the code here. I was just repeating the existing exit pattern.

Reply:
I do not see this pattern you are referring to. Please exit-out cleanly here.

If you are talking about the `exit` on line 892, please take a closer look. That exit is from a forked child process.

Reply:
I had already changed the exit method at the time of this comment. How do you feel about the current code?